src/main/java/com/adyen/Client.java

```diff
@@ -13,6 +13,10 @@ public class Client {
     public static final String LIB_VERSION = "32.0.0";
     public static final String TERMINAL_API_ENDPOINT_TEST = "https://terminal-api-test.adyen.com";
     public static final String TERMINAL_API_ENDPOINT_LIVE = "https://terminal-api-live.adyen.com";
+    public static final String ENDPOINT_TERMINAL_CLOUD_LIVE = "https://terminal-api-live.adyen.com";
```

Comment:
`"https://terminal-api-live.adyen.com"` has been added twice (see `TERMINAL_API_ENDPOINT_LIVE`)

Reply:
Perhaps moving these endpoints-names out of the Client into the respective terminal-api service or its own config would be better, what do you think?

Reply:
Yes that makes sense. I'll remove the duplicate. about moving the endpoint names to a separate config. I'm not sure I see that for .Net "https://terminal-api-live.adyen.com" is added as CloudApiEndPointEULive in the clientconfig. https://github.com/Adyen/adyen-dotnet-api-library/blob/af0f3b7259736e961992840e7b9514befdcc6789/Adyen/Constants/ClientConfig.cs#L9C28-L9C52 but for php it is like you suggested for Java that EU is the default live endpoint https://github.com/Adyen/adyen-php-api-library/blob/af262a702228c473bbb4c00f9d7000f0d85ad7e5/src/Adyen/Client.php#L30C41-L30C80. Do you suggest creating a separate terminalconfig for all the libraries and set a specific ENDPOINT_TERMINAL_CLOUD_EU_LIVE?

Reply:
Keep the current strings as is, and default to EU